pkg/utils/finalizer_utils.go

```diff
@@ -32,3 +37,17 @@ func RemoveString(slice []string, str string) (result []string) {
 	}
 	return result
 }
+
+func IsPvcOrphan(zkPvcName string, replicas int32) bool {
```

Comment:
The acronym should have common capitalization here: `IsPVCOrphan`.

Reply:
Ok. Sure.

Reply:
Done